doc/devel/release_guide.rst

```diff
@@ -380,8 +381,12 @@ PyPI. Most builders should trigger automatically once the tag is pushed to GitHu
 
 .. _release_upload_bin:
 
-Make distribution and upload to PyPI
-====================================
+Manually uploading to PyPI
+==========================
+
+As noted above, the GitHub Actions workflow should build and upload source tarballs and
+wheels automatically. If for some reason, you need to upload these artifacts manually,
+then follow the instructions in this section.
```

Comment:
Use an `.. info::` or `.. warning::` admonition to set this meta-information apart visually.